src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+
+* {
```

Comment:
Using the universal selector '*' is not recommended as it impacts performance. Instead, apply styles to specific elements that require them.